package.json

```diff
@@ -14,6 +14,7 @@
   "dependencies": {
     "aphrodite": "^0.3.0",
     "jquery": "^2.1.1",
+    "katex": "0.9.0",
```

Comment:
This wasn't an issue before b/c we were using the global version of katex until recently.